tests/integration/test_upstream.py

```diff
@@ -25,7 +25,7 @@ def test_get_current_version(upstream_instance):
 
 
 @pytest.mark.parametrize(
-    "m_v,exp", (("1.1000.1000000", "1.1000.1000000"), (None, "0.1.0"))
+    "m_v,exp", (("1.1000.1000000", "1.1000.1000000"), (None, "0.1.0"), ("0.0.3", "0.1.0"), ("176", "176"))
```

Comment:
what if the version is "kfm23lkfmn23lkfmn23klfnm23"?

Reply:
In that case, LegacyVersion is used https://packaging.pypa.io/en/latest/version/#packaging.version.LegacyVersion.

Example:
```python3
from packaging import version
>>> version.parse('abrakadabra') > version.parse("kfm23lkfmn23lkfmn23klfnm23")
False
```

Reply:
neat!